lib/matplotlib/tests/test_font_manager.py

```diff
@@ -31,6 +31,20 @@ def test_font_priority():
     assert cmap[8729] == 30
 
 
+def test_ttflist_weight():
+    assert all(isinstance(f.weight, six.string_types)
+               for f in fontManager.ttflist)
+
+
+def test_score_weight():
+    assert (0 ==
+            fontManager.score_weight("regular", "regular") ==
+            fontManager.score_weight("bold", "bold") <
+            fontManager.score_weight("normal", "regular") ==
+            fontManager.score_weight(400, 400) <
```

Comment:
This is because "normal" and "regular" have the same numerical weight (https://github.com/matplotlib/matplotlib/blob/e69ea36995a76c37b928f9cab9af1161f8d36e05/lib/matplotlib/font_manager.py#L99-L100).